src/types.ts

```diff
@@ -190,6 +190,7 @@ export interface ExcalidrawProps {
     appState: AppState,
   ) => JSX.Element;
   UIOptions?: UIOptions;
+  detectNearestScroll?: boolean;
```

Comment:
I'd name this `detectScrollChange` as `nearestScroll` isn't descriptive + it's an implementation detail. Though maybe the latter part was on purpose to indicate it's not a 100% surefire solution? Even so, I'd note that in readme and opt-out for more clear prop naming.

Reply:
Yes since we are only handling nearest scroll so wanted to make it clear via the prop name.